reference/motor.md

```diff
@@ -311,18 +315,22 @@ wb_motor_set_velocity(tag, desired_speed);  // rad/s
 The `wb_motor_get_target_position` function allows the user to get the target position.
 This value matches with the argument given to the last `wb_motor_set_position` function call.
 
-The `wb_motor_set_velocity` function specifies the velocity that motor should reach while moving to the target position.
+The `wb_motor_set_velocity` function specifies the velocity that a motor should reach while moving to the target position.
+The velocity is expressed in *radian per second* for rotational motors and in *meter per second* for linear motors.
```

Comment:
to be consistent we should maybe add the abbreviation also in case of the position, i.e. `(rad/s)` and `(m/s)`?
personally I found that the symbol is more immediate to understand than the extended text

same at line 30, 64, 270, 327